hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/RepositoryApplicationConfiguration.java

```diff
@@ -390,7 +390,8 @@ AfterTransactionCommitExecutorHolder afterTransactionCommitExecutorHolder() {
     public MethodValidationPostProcessor methodValidationPostProcessor() {
         final MethodValidationPostProcessor processor = new MethodValidationPostProcessor();
         processor.setValidator(Validation.byDefaultProvider().configure()
-                .addProperty(HibernateValidatorConfiguration.ALLOW_PARALLEL_METHODS_DEFINE_PARAMETER_CONSTRAINTS,
+                .addProperty(BaseHibernateValidatorConfiguration.ALLOW_PARAMETER_CONSTRAINT_OVERRIDE, "true")
```

Comment:
Just out of curiosity why did `ALLOW_PARAMETER_CONSTRAINT_OVERRIDE` get added here too?

Reply:
It is indeed not needed. I removed it.